plugins/okta/pkg/okta/okta.go

```diff
@@ -138,7 +140,7 @@ func (oktaPlugin *Plugin) Info() *plugins.Info {
 		Name:        "okta",
 		Description: "Okta Log Events",
 		Contact:     "github.com/falcosecurity/plugins/",
-		Version:     "0.11.0",
+		Version:     "0.11.1",
```

Comment:
```suggestion
		Version:     "0.11.0",
```

Hey @SKosier 

0.11 has not been released yet.